### PR TITLE
Update ThreeWindingsTransformer & TwoWindingsTransformer documentation

### PR DESCRIPTION
**Pull request by tadam50**
Signed-off-by: Thomas ADAM <tadam@silicom.fr>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
'#1144'

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [x] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

### CODE REVIEW

File: docs/iidm/model/threeWindingsTransformer.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Three windings transformers can also have [current limits](currentLimits.md) def
 | G | double | S | yes | - | The nominal magnetizing conductance specified at the voltage of the leg |
 | B | double | S | yes | - | The nominal magnetizing susceptance specified at the voltage of the leg |
 | RatedU | double | kV | yes | - | The rated voltage |
+| RatedS | double | kVA | no | - | The apparent power |
```
Comment (annetill) [inline docs/iidm/model/threeWindingsTransformer.md]: I am not sure about the multiplier : I think MVA is better.

Comment (annetill) [reply docs/iidm/model/threeWindingsTransformer.md]: And can you add as a note that for the leg of a three widnings transformer, the normal apparent power shall be identical at both sides (network side and star bus side).

Comment (annetill) [reply docs/iidm/model/threeWindingsTransformer.md]: I prefer normal apparent power than apparent power only.

File: docs/iidm/model/twoWindingsTransformer.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ A [ratio tap changer](./ratioTapChanger.md) and/or a [phase tap changer](./phase
 | $$b_{nom}$$ | double | S | yes | - | The nominal magnetizing susceptance at the side 2 of the transformer |
 | $$V_{1\ nom}$$ | double | kV | yes | - | The rated voltage at side 1 |
 | $$V_{2\ nom}$$ | double | kV | yes | - | The rated voltage at side 2 |
+| ratedS | double | kVA | no | - | The apparent power |
```
Comment (annetill) [inline docs/iidm/model/twoWindingsTransformer.md]: Same remarks: I think it is MVA, I prefer to normal apparent power and if you can add a note: "for a two windings transformer, the normal apparent power shall be identical at both sides 1 and 2.

File: docs/iidm/model/threeWindingsTransformer.md (hunk 1 above)
Comment (MioRtia) [reply docs/iidm/model/threeWindingsTransformer.md]: > I am not sure about the multiplier : I think MVA is better.

The javadoc should probably get changed too.

File: docs/iidm/model/threeWindingsTransformer.md [hunk 3]
```diff
@@ -45,6 +45,9 @@ Three windings transformers can also have [current limits](currentLimits.md) def
 | G | double | S | yes | - | The nominal magnetizing conductance specified at the voltage of the leg |
 | B | double | S | yes | - | The nominal magnetizing susceptance specified at the voltage of the leg |
 | RatedU | double | kV | yes | - | The rated voltage |
+| RatedS | double | MVA | no | - | The normal apparent power |
+
+For the leg of a three widnings transformer, the normal apparent power shall be identical at both sides (network side and star bus side).
```
Comment (annetill) [inline docs/iidm/model/threeWindingsTransformer.md]: typo : windings

**Comment by annetill** [comment]
Thanks for your documentation !

**Pull request merged**